trunk/SUAVE/Methods/Missions/Segments/Common/Energy.py

```diff
@@ -82,4 +133,71 @@ def update_thrust(segment):
     conditions.weights.vehicle_mass_rate       = results.vehicle_mass_rate
     
 
-    
\ No newline at end of file
+## @ingroup Methods-Missions-Segments-Common
+def update_battery(segment):
+    """ Evaluates the energy network to find the thrust force and mass rate
+
+        Inputs -
+            segment.analyses.energy_network    [Function]
+
+        Outputs -
+            state.conditions:
+               frames.body.thrust_force_vector [Newtons]
+               weights.vehicle_mass_rate       [kg/s]
+
+
+        Assumptions -
+
+
+    """    
+    
+    # unpack
+    energy_model = segment.analyses.energy
+
+    # evaluate
+    results   = energy_model.evaluate_thrust(segment.state)
+
+def update_battery_age(segment):  
+    """This is an aging model for 18650 lithium-nickel-manganese-cobalt-oxide batteries. 
```

Comment:
Is this really only for LiNiMnCO? If so, it looks like this can be applied to others the way it is setup

Reply:
This is only for NMC cells.

Reply:
Battery aging is also cell specific

Reply:
So then maybe this should go as a function in the NMC subclass of Lithium_Ion instead of here; otherwise it appears as a function that can be called by the user on any battery even though it is designed specifically for the NMC

Reply:
I agree make a method for the battery and make this an arbitrary script that calls the method from the battery type.

This script is made to point to other scripts